proposals/16:upgrade-8/use.sh

```diff
@@ -22,7 +22,7 @@ waitForBlock 2
 waitForBlock 3
 # fund provision pool
 stakeamount="20000000${USDC_DENOM}"
-agd tx bank send "validator" "agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346" "$stakeamount" -y --keyring-backend=test --chain-id="$CHAINID"
+agd tx bank send "validator" "agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346" "$stakeamount" -y --keyring-backend=test --chain-id="$CHAINID" -bblock
```

Comment:
Q: why are all `-bblock` added right after `agd tx` except for this one added to the end ?

Reply:
I'll merge but leave this unresolved for @michaelfig to see

Reply:
Yeah, this inconsistency just goes to show how many changes I had to make manually.  No particular reason behind it.

As for always adding `-bblock` right after `agd tx`, that's generally so subsequent `-b<FLAG>` arguments supplied by callers can override the initial `-bblock`.